Formula/magnetix.rb

```diff
@@ -7,6 +7,14 @@ class Magnetix < Formula
 
   depends_on :xcode => :build
 
+  # Port audio code from QTKit to AVFoundation
+  # Required since 10.12 SDK no longer includes QTKit.
+  # Submitted by email to the developer.
+  patch do
```

Comment:
Given it's a major path, I'd rather not end up shipping two different versions of the audio code - would make support easier if we're only supporting one version.